podcasts/CustomStepper.swift

```diff
@@ -0,0 +1,188 @@
+import UIKit
+
+class CustomStepper: UIControl {
```

Comment:
Not proud of copying/pasting `CustomTimeStepper` here. I tried playing with generics but forgot that Storyboard only plays nice with Obj-C. 🤡